File: EulerProblemOne.java
import java.util.ArrayList;

public class EulerProblemOne {
	private static int cap = 1000;
	private static int rThree;
	private static int rFive;
	private static ArrayList<Integer> nums = new ArrayList<Integer>();
	private static int sum = 0;
	public static void main(String[] args){
		for(int i = 0; i <cap; i++){
			rThree = findMultThree(i);
			rFive = findMultFive(i);
			if(rThree == 0 || rFive ==0 ){
				nums.add(i);
			}
		}
		for(int i = 0; i< nums.size(); i++){
			sum += nums.get(i);
			}
		System.out.println(sum);
			
		}
	
public static int findMultThree(int i){
	int r = i%3;
	return r;
	}
public static int findMultFive(int i){
	int r = i%5;
	return r;
	}
}

File: EulerProblemOne.py
#Molly Depew
#Euler problem 1
#Find the sum of all the multiples of 3 and 5 under 1000

def main():
    cap = 1000
    nums = []
    for k in range(cap):
        if findMultThree(k)== 0:
            nums.append(k)
        elif findMultFive(k) == 0:
            nums.append(k)
    ans = sum(nums)
    print(ans)
    
  
# modulus functions. if the functions return anything other than 0
# the integer is not a multiple of 5 or 3
def findMultThree(i):
    r = i%3
    return r
def findMultFive(i) :
    r = i%5
    return r
        
main()
    
    
    
